contrib/devtools/update-translations.py

```diff
@@ -159,6 +161,11 @@ def postprocess_translations(reduce_diff_hacks=False):
                 for translation in translations:
                     if translation is None:
                         continue
+
+                    # Check if btc address is included in translation after removing whitespaces
+                    if re.search(ADDRESS_RE, translation.replace(' ', '')):
```

Comment:
To remove all whitespaces:
```py
translation.translate( { ord(c):None for c in ' \n\t\r' } )
```
ot
```py
''.join(translation.split())
```

Reply:
done..the implementation is changed in accordance with @Empact 's suggestion